### PR TITLE
Layout/destination

### PR DESCRIPTION
**Pull request by 95Rawan**
Created Destination page
 * Created Destination class component that has react's picker component.
* Put fake data to each picker until we get the database queries ready.
* Created actions for each item's picker to handle the value change of each one.
* Created a case for each action in the reducer to handle it and update the state.
* Created an onButtonPress function to navigate to the next screen."

### CODE REVIEW

File: client/Sawaah/src/Components/SharedComponent/NavBar/index.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ const NavBar = () => {
     <NavBarWrapper>
       <NavBarLogoWrapper>
         <NavBarLogo
-          source={require('./../../../assets/images/airplane-logo.png')}
+          source={require('s./../../../assets/images/airplane-logo.png')}
```
Comment (FarahZaqout) [inline client/Sawaah/src/Components/SharedComponent/NavBar/index.js]: this is not right

File: client/Sawaah/src/actions/index.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+export const countrySelector = country => {
+  return {
+    type: 'SELECT_COUNTRY',
+    country,
```
Comment (FarahZaqout) [inline client/Sawaah/src/actions/index.js]: best practice is to call these ```payload```.
We are already describing the action with the action type. Using the name ```payload``` makes it less confusing as the app gets bigger.

File: client/Sawaah/src/Components/SharedComponent/NavBar/index.js (hunk 1 above)
Comment (95Rawan) [reply client/Sawaah/src/Components/SharedComponent/NavBar/index.js]: fixed it

File: client/Sawaah/src/actions/index.js (hunk 2 above)
Comment (95Rawan) [reply client/Sawaah/src/actions/index.js]: changed it.

**Pull request merged**